_posts/2024-02-07-#28950.md

```diff
@@ -5,19 +5,60 @@ date: 2024-02-07
 title: "Add `maxfeerate` and `maxburnamount` args to submitpackage"
 pr: 28950
 authors: [instagibbs]
-components: ["rpc"]
+components: ["rpc/rest/zmq"]
```

Comment:
In the future I'd recommend using the new post [script](https://github.com/bitcoin-core-review-club/website/blob/master/hosting.md#making-a-new-post), as it would scrape the labels and prevent things like #744.